languages/python/oso/polar/ffi.py

```diff
@@ -127,7 +127,7 @@ def __del__(self):
     def call_result(self, call_id, value):
         """Make an external call and propagate FFI errors."""
         if value is None:
-            value = ffi.NULL
+            value = to_c_str("null")
```

Comment:
Yeah. On the Rust side, `value` is an optional value. We were previously doing a bit of weird logic in the polar-c-api to check if `value == nil` then make it `None`, otherwise deserialize.

But `null` deserializes to `None`, so this handles this directly.

Oh, `None` already serialializes to "null", so this if can be removed entirely.